doc/source/ray-observability/key-concepts.rst

```diff
@@ -147,58 +115,34 @@ Let's look at a code example to see how this works.
-- ``job-driver-[submission_id].log``: The stdout of a job submitted via the :ref:`Ray Jobs API <jobs-overview>`.
-- ``log_monitor.[log|err]``: The log monitor is in charge of streaming logs to the driver. ``log.`` file contains logs generated from the log monitor's logger. ``.err`` file contains the stdout and stderr printed from the log monitor. They are usually empty except when the log monitor crashes unexpectedly.
-- ``monitor.[out|err]``: Stdout and stderr of a cluster launcher.
-- ``monitor.log``: Ray's cluster launcher is operated with a monitor process. It also manages the autoscaler.
-- ``plasma_store.[out|err]``: Deprecated.
-- ``python-core-driver-[worker_id]_[pid].log``: Ray drivers consist of CPP core and Python/Java frontend. This is a log file generated from CPP code.
-- ``python-core-worker-[worker_id]_[pid].log``: Ray workers consist of CPP core and Python/Java frontend. This is a log file generated from CPP code.
-- ``raylet.[out|err]``: A log file of raylets.
-- ``redis-shard_[shard_index].[out|err]``: Redis shard log files.
-- ``redis.[out|err]``: Redis log files.
-- ``runtime_env_agent.log``: Every Ray node has one agent that manages :ref:`runtime environment <runtime-environments>` creation, deletion and caching.
-  This is the log file of the agent containing logs of create/delete requests and cache hits and misses.
-  For the logs of the actual installations (including e.g. ``pip install`` logs), see the ``runtime_env_setup-[job_id].log`` file (see below).
-- ``runtime_env_setup-[job_id].log``: Logs from installing :ref:`runtime environments <runtime-environments>` for a task, actor or job.  This file will only be present if a runtime environment is installed.
-- ``runtime_env_setup-ray_client_server_[port].log``: Logs from installing :ref:`runtime environments <runtime-environments>` for a job when connecting via :ref:`Ray Client <ray-client-ref>`.
-- ``worker-[worker_id]-[job_id]-[pid].[out|err]``: Python or Java part of Ray drivers and workers. All of stdout and stderr from tasks or actors are streamed here. Note that job_id is an id of the driver.- 
+
+By default, all stdout and stderr of tasks and actors are redirected to the driver output. View :ref:`Configuring Logging <log-redirction-to-driverg>` for more details.
+
+
+
+Job logs
+~~~~~~~~
+Ray applications are usually run as Ray Jobs. Worker logs of Ray jobs are always captured in the Ray logging directory while Driver logs are not.
+
+Only if the Ray job is submitted via :ref:`Ray Jobs API <jobs-quickstart>`, the driver log is captured in the logging direcotry and available from the dashboard, CLI (using the ``ray job logs`` :ref:`CLI command <ray-job-logs-doc>`), or the :ref:`Python SDK <ray-job-submission-sdk-ref>` (``JobSubmissionClient.get_logs()`` or ``JobSubmissionClient.tail_job_logs()``).
```

Comment:
```suggestion
Only if the Ray Job is submitted with the :ref:`Ray Jobs API <jobs-quickstart>`, is the driver log captured in the logging direcotry and available from the dashboard, CLI (using the ``ray job logs`` :ref:`CLI command <ray-job-logs-doc>`), or the :ref:`Python SDK <ray-job-submission-sdk-ref>` (``JobSubmissionClient.get_logs()`` or ``JobSubmissionClient.tail_job_logs()``).
```

Reply:
This sentence was difficult for me to parse with the two `or`s at the end of the sentence.